src/service/resource.js

```diff
@@ -678,7 +678,7 @@ export class Resource {
    * @return {boolean}
    */
   isDisplayed(usePremeasuredRect = false) {
-    devAssert(!usePremeasuredRect || !this.intersect_);
+    devAssert(!usePremeasuredRect || this.intersect_);
```

Comment:
Is the point to XOR them? Meaning, is it valid to have `usePremeasureRect = false` and `intersect = true`?

Reply:
Yea that's a valid case. We want `usePremeasuredRect -> intersect_` logically, I just messed it up.